### ISSUE TITLE
Ensure custom data is included even with errors

### ISSUE THREAD

**Issue by rokob** [rollbar/rollbar-php#207]
Supposedly the extra custom data is ignored in calls with errors, including this one from the docs:

```php
Rollbar::log(Level::error(), $e, array("my" => "extra", "data" => 42));
```

Figure out if this is true and fix this. If it is not true, ensure that there is a test for it, but also try to figure out if this was true in within the past version and where it was fixed.

**Comment by ArturMoczulski** [comment]
I tracked this down all the way v1.0.0 and also investigated the closest functionality there is to this in v0.18.2. It all works as expected - $extra is included in the payload. PR contains a detailed test to ensure this as well.

@rokob do you know where the suspicion of this not working correctly came from?

**Comment by rokob** [comment]
A user report, but it was limited in detail so this is probably sufficient.

**Comment by ytn3rd** [comment]
This issue seems to have popped up again. Custom data is not included when logging.

```
Rollbar::log(Level::ERROR, $e, array("my" => "extra", "data" => 42));
```

Error is logged, but in no where in the request does it log any of these other parameters.

**Comment by ArturMoczulski** [comment]
Interesting. Can you configure your `Rollbar` class with `verbosity` at `\Psr\Log\LogLevel::DEBUG` and post your `/tmp/rollbar.debug.log` (please, scrub your access token)?

**Comment by ytn3rd** [comment]
TLDR: Everything magically seems to work now...

So I enabled verbosity, and debugged with this, 
```
Rollbar::log(Level::EMERGENCY, 'This is Level::EMERGENCY', array("my" => "extra0", "data" => 40));
Rollbar::log(Level::ALERT, 'This is Level::ALERT', array("my" => "extra1", "data" => 41));
Rollbar::log(Level::CRITICAL, 'This is Level::CRITICAL', array("my" => "extra2", "data" => 42));
Rollbar::log(Level::ERROR, 'This is Level::ERROR', array("my" => "extra3", "data" => 43));
Rollbar::log(Level::WARNING, 'This is Level::WARNING', array("my" => "extra4", "data" => 44));
Rollbar::log(Level::NOTICE, 'This is Level::NOTICE', array("my" => "extra5", "data" => 45));
Rollbar::log(Level::INFO, 'This is Level::INFO', array("my" => "extra6", "data" => 46));
Rollbar::log(Level::DEBUG, 'This is Level::DEBUG', array("my" => "extra7", "data" => 47));
```
Everything came through (although I never got my log output anywhere). I signed into our system so our session data was also logged, everything came through as expected.

It did log the my data stuff as message.my, custom.my, message.data and custom.data. So it was duplicated in there.

I tried with a try/catch

```
try 
{
    throw new Exception('This is via an exception');
}
catch (Exception $err)
{   
    Rollbar::log(Level::ERROR, $err, array("my" => "extra3", "data" => 43));
}
```

Everything worked... Although this time. my data was only in custom.my and custom.data, there is no message.my or message.data.

Still on version 1.3.6. I have no words, but am happy its working at least.
